### PR TITLE
Use oldstyle server DID when setting DNS _did

### PR DESCRIPTION
**Pull request by expede**
Still need to write a couple tests to make sure that the format is _actually_ what we expect, but in theory this is done

### CODE REVIEW

File: fission-web-server/library/Fission/Web/Server/Types.hs [hunk 1]
```diff
@@ -468,7 +470,7 @@ instance Server.DID.Publicize Server where
       ourURL         = URL (URL.DomainName . Text.pack $ baseUrlHost host) Nothing
       txtRecordURL   = URL.prefix' (URL.Subdomain "_did") ourURL
 
-      txtRecordValue = textDisplay did
+      txtRecordValue = textDisplay $ DID.Oldstyle did
```
Comment (expede) [inline fission-web-server/library/Fission/Web/Server/Types.hs:473]: This is the main event

File: .github/workflows/artifacts-nix.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os:
-          - macos-latest
+          # - macos-latest
```
Comment (expede) [inline .github/workflows/artifacts-nix.yaml:16]: Just fighting with GH Actions right now

File: stack.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-resolver: lts-17.11
+resolver: lts-17.12
```
Comment (expede) [inline stack.yaml:1]: Damnit Stackage, stop releasing new things 🤣

**Review by walkah — APPROVED** [review]
Seth Meyers Thank You - 26gsjCZpPolPr3sBy
Will Ferrell Yes - 3ohzdIuqJoo8QdKlnW

**Pull request merged**